controller/instance_handler.go

```diff
@@ -337,11 +339,13 @@ func (h *InstanceHandler) ReconcileInstanceState(obj interface{}, spec *longhorn
 }
 
 func (h *InstanceHandler) printInstanceLogs(instanceName string, obj runtime.Object) error {
-	stream, err := h.instanceManagerHandler.LogInstance(obj)
+	ctx, cancel := context.WithCancel(context.TODO())
+	defer cancel()
+	client, stream, err := h.instanceManagerHandler.LogInstance(ctx, obj)
 	if err != nil {
 		return err
 	}
-	defer stream.Close()
+	defer client.Close()
```

Comment:
Should we close the returned stream as well?

Reply:
Stream gets automatically closed when client is closed.
For knowledge sharing REF: https://github.com/longhorn/longhorn-instance-manager/blob/60cd930107ef19273b9c490814b69a59669e306f/vendor/google.golang.org/grpc/stream.go#L127-L143